x/concentrated-liquidity/incentives.go

```diff
@@ -240,11 +250,21 @@ func calcAccruedIncentivesForAccum(ctx sdk.Context, accumUptime time.Duration, l
 		}
 
 		// Total amount emitted = time elapsed * emission
-		totalEmittedAmount := timeElapsed.Mul(incentiveRecordBody.EmissionRate)
+		totalEmittedAmount := timeElapsed.MulTruncate(incentiveRecordBody.EmissionRate)
```

Comment:
to future proof this a bit more, thoughts on adding comments in line here for the expected range of each term?

Reply:
Attempted to address here:
[9e10789](https://github.com/osmosis-labs/osmosis/pull/7409/commits/9e10789efcd423e49320cb9e8389067b28d84cc0)

Let me know what you think